contracts/contracts/cairo/validator/Price_Consumer.cairo

```diff
@@ -0,0 +1,71 @@
+%lang starknet
```

Comment:
Love the example, but it should probably be moved to examples: [examples/contracts/aggregator-consumer/](https://github.com/smartcontractkit/chainlink-starknet/tree/develop/examples/contracts/aggregator-consumer)